geode-core/src/test/java/org/apache/geode/management/internal/configuration/converters/RegionConverterTest.java

```diff
@@ -91,8 +100,27 @@ public void fromConfig() throws Exception {
     assertThat(config.getName()).isEqualTo("test");
     assertThat(config.getType()).isEqualTo("REPLICATE");
     RegionAttributesType regionAttributes = config.getRegionAttributes();
+    assertThat(regionAttributes.getDataPolicy()).isEqualTo(RegionAttributesDataPolicy.REPLICATE);
     assertThat(regionAttributes.getKeyConstraint()).isEqualTo("bar");
     assertThat(regionAttributes.getValueConstraint()).isEqualTo("foo");
     assertThat(regionAttributes.getDiskStoreName()).isEqualTo("diskstore");
   }
+
+  @Test
+  public void checkDefaultRegionAttributesForShortcuts() throws Exception {
+    URL xmlResource = RegionConverterTest.class.getResource("RegionConverterTest.xml");
```

Comment:
If is not clear to me how valuable this test is. It seems like we just duplicate everything that is in our management code that defines a type into the RegionConverterTest.xml. The xml sets both the refid and the nested region attributes.
Maybe this is worth testing.
But what seems more valuable, and what is not tested, is that the attributes returned from "createRegionAttributesByType" and the same as what the core defines for each RegionShortcut.

Reply:
this xml was originally created by using the method in cache and saved as a master xml.